src/maestro/cli/fastapi_serve.py

```diff
@@ -58,6 +59,13 @@ def __init__(self, agents_file: str, agent_name: Optional[str] = None):
             description="HTTP API for serving Maestro agents",
             version="1.0.0",
         )
+        self.app.add_middleware(
+            CORSMiddleware,
+            allow_origins=["*"],
+            allow_credentials=True,
+            allow_methods=["*"],
+            allow_headers=["*"],
```

Comment:
I think ideally there would be config options for these in the agent spec somehow, with defaults in case they aren't specified

Reply:
would it be better to config them in the env instead? I'm unsure if adding serve configs to the agent is the best solution. Also which values should have configs? just `allow_origins` or all four of them?.

Reply:
We could probably just start with allow_origins... if others are needed we should be able to build off that approach (and it lets us add user knobs slowly only if there's need)

As for agent spec vs. envvars, I don't really have a strong opinion either way.

Reply:
maybe something like:

`enable_cors: true`

And false by default or w/o specifying

Reply:
I'm making an env var list that leave it disable when unset, currently testing it